src/locales/en-US.json

```diff
@@ -1266,6 +1266,7 @@
       "successDisableBmcShell": "Successfully disabled BMC shell (via SSH), Please allow 15 seconds for the disable to become effective.",
       "successEnableBmcShell": "Successfully enabled BMC shell (via SSH)",
       "successNetworkPolicyUpdate": "Successfully updated %{policy}.",
+      "successIpmiNetworkPolicyUpdate": "The BMC normally takes about 30 seconds to change the %{policy} state.",
```

Comment:
Can this be "Successfully updated %{policy}. The BMC normally takes about 30 seconds to change the state."